Upstream: No
Reason: install is verbose but only last line should be tested
--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -1117,4 +1117,4 @@
     print(job.stdout)
     print(job.stderr)
     assert job.returncode == 0
-    assert job.stdout.rstrip() == "2024.10.9"
+    assert job.stdout.rstrip().splitlines()[-1] == "2024.10.9"
